src/node/build-node.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o pipefail
+set -e
```

Comment:
This should make sure the working directory is correct like other scripts with `cd "$(dirname "$0")"`